Update linux.txt

baldingTomato opened the pull request:
Edit for rerun

baldingTomato committed:
Update linux.txt

ziobron commented:
@baldingTomato, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

baldingTomato committed:
Added numbers of lines in grep command

The first time i did not understand that i have to use 'grep -n ...' and then paste it to the file.

ziobron commented:
✅ CI Passed - Task: linux
New user detected (@baldingTomato).  Trying to automatically map accounts.

ziobron commented:
✅ CI Passed - Task: linux
🏅 Granted 10 XP for @baldingTomato
